x-pack/legacy/plugins/apm/public/projections/transaction_groups.ts

```diff
@@ -5,39 +5,51 @@
  */
 
 import { Setup } from '../../server/lib/helpers/setup_request';
```

Comment:
It just occurs to me that the `projections` folder is located on the client. This seems like a mistake?

Reply:
Yeah, I thought `public` was the shared folder, and `common` was some legacy stuff. 😁

Reply:
Ah, yeah. The naming could be more clear.